qemu/tests/win_virtio_update.py

```diff
@@ -36,6 +36,17 @@ def check_cdrom(timeout):
                 break
         return vols
 
+    def virtio_driver_install(session, cmd, driver):
+        driver_install_timeout = int(params.get('driver_install_timeout', 720))
+        fail_flag = False
+        fail_logs = ""
+        try:
+            session.cmd(cmd, timeout=driver_install_timeout)
+        except Exception, msg:
+            fail_flag = True
+            fail_logs += "driver %s install failed. Msg %s" % (driver, msg)
+            error.context("Install drivers %s failed " % driver, logging.info)
```

Comment:
@suqinhuang It is a prompts the driver is failed to install in logs and then start installing next driver